python/ray/data/tests/test_execution_optimizer.py

```diff
@@ -991,11 +995,10 @@ def test_from_huggingface_e2e(ray_start_regular_shared, enable_optimizer):
     assert ray.get(ray_dataset.to_arrow_refs())[0].equals(data["train"].data.table)
     assert len(ray_dataset.take_all()) > 0
 
-    # Check that metadata fetch is included in stats.
+    # Check that metadata fetch is included in stats;
+    # the underlying implementation uses the `FromArrowRefs` operator.
     assert "FromArrowRefs" in ray_dataset.stats()
-    # Underlying implementation uses `FromArrowRefs` operator
     assert ray_dataset._plan._logical_plan.dag.name == "FromHuggingFace"
-    _check_usage_record(["FromHuggingFace"])
```

Comment:
hmm why it's not caught as part of CI in https://github.com/ray-project/ray/pull/32959 ?